src/index.js

```diff
@@ -1,56 +1,21 @@
+import ToDo from './todo.js';
+import addEventsDragAndDrop from './drag-and-drop.js';
+import Task from './task.js';
```

Comment:
* As stated in the [Project Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Add%202%20new,completed%3A%20true%20/%20false) a module should be imported for handling status update eg: (completed: true/false).